data/json/items/armor/arm_prosthetics.json

```diff
@@ -0,0 +1,162 @@
+  },
+  {
+    "id": "right_arm_neuroprosthetic",
+    "type": "ARMOR",
+    "name": { "str": "neuroprosthetic right arm" },
+    "description": "A neuroprosthetic arm, used as a replacement limb for those missing one.  Its modern design and sleek ergonomics make it very comfortable and highly suitable for use, and advanced designing makes it look exactly like the original.  As a bleeding edge design, this one relies on wires attached to the forehead to provide motor input, allowing the arm to be operated just like the original limb, a precious commodity in these times.  This one is for the right arm.",
+    "weight": "2000 g",
+    "volume": "8 L",
+    "longest_side": "50 cm",
+    "price": 750000,
+    "price_postapoc": 150000,
+    "to_hit": 1,
+    "material": [ "steel", "plastic" ],
+    "symbol": "|",
+    "color": "brown",
+    "warmth": 15,
+    "material_thickness": 1,
+    "flags": [ "ARM_PROSTHETIC", "ONLY_ONE" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "RIGHT_NEUROPROSTHETIC_ARM" ] } ] },
+    "armor": [ { "encumbrance": 0, "coverage": 100, "covers": [ "arm_stub_r", "arm_neuroprosthetic_r", "hand_neuroprosthetic_r" ] } ]
+  },
+  {
+    "id": "left_arm_neuroprosthetic",
+    "type": "ARMOR",
+    "name": { "str": "neuroprosthetic left arm" },
+    "description": "A neuroprosthetic arm, used as a replacement limb for those missing one.  Its modern design and sleek ergonomics make it very comfortable and highly suitable for use, and advanced designing makes it look exactly like the original.  As a bleeding edge design, this one relies on wires attached to the forehead to provide motor input, allowing the arm to be operated just like the original limb, a precious commodity in these times.  This one is for the left arm.",
+    "weight": "2000 g",
+    "volume": "8 L",
+    "longest_side": "50 cm",
+    "price": 750000,
+    "price_postapoc": 150000,
+    "to_hit": 1,
+    "material": [ "steel", "plastic" ],
+    "symbol": "|",
+    "color": "brown",
+    "warmth": 15,
+    "material_thickness": 1,
+    "flags": [ "ARM_PROSTHETIC", "ONLY_ONE" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "LEFT_NEUROPROSTHETIC_ARM" ] } ] },
+    "armor": [ { "encumbrance": 0, "coverage": 100, "covers": [ "arm_stub_l", "arm_neuroprosthetic_l", "hand_neuroprosthetic_l" ] } ]
+  },
+  {
+    "id": "right_arm_prosthetic_wood",
+    "type": "ARMOR",
+    "name": { "str": "wooden prosthetic right arm" },
+    "description": "A prosthetic arm, used as a replacement limb for those missing one.  Dating back to antiquity, this wooden arm was often used by many individuals as a simple way to hide the loss of one of their limbs.  Outside of that, it's rigidity and lack of flexibility make it nearly useless for practical purposes, though it can help with very basic tasks, like lifting objects.  This one is for the right arm.",
```

Comment:
```suggestion
    "description": "A prosthetic arm, used as a replacement limb for those missing one.  Dating back to antiquity, this wooden arm was often used by many individuals as a simple way to hide the loss of one of their limbs.  Outside of that, its rigidity and lack of flexibility make it nearly useless for practical purposes, though it can help with very basic tasks, like lifting objects.  This one is for the right arm.",
```